src/App.js

```diff
@@ -1,42 +1,134 @@
 import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
-import TodoListItem from './TodoListItem';
+// import TodoListItem from './TodoListItem';
+// import { json } from 'express';
 
 
 
 function App() {
  
   const [todoList, setTodoList] = React.useState([]);
   const [isLoading, setIsLoading] = React.useState(true);
+
+
+  const fetchData = async() => {
+      const options = {
+        method: "GET", 
+        headers: {
+          Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`
+        }
+      };
+      const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+      
+      console.log(url);
+      console.log("test",options);
+
+
+        try {
+          const response = await fetch(url, options);
+
+          if (!response.ok) {
+          const message = `Error: ${response.status}`;
+          throw new Error(message);
+          }
+        const data = await response.json();
+
+      console.log("DATA",data);
+
+      const todos = data.records.map((todo) => {
+
+        const newTodo = {
+          id: todo.id,
+          title: todo.fields.title
+        }
+        return newTodo
+      });
+        
+      setTodoList(todos);
+      setIsLoading(false);
+     
+      // console.log(data);
+      return data;
+        }
+        catch {
+          console.log("error messange");
```

Comment:
This one's ok to keep, since it's particularly requested by the assignment, but eventually we would replace this code with something that renders the error on the page instead

Reply:
Fixed